tests/plugin/test_override_by_external_plugins.py

```diff
@@ -37,15 +23,7 @@ def test_disabled_plugin_is_not_executed(
 
     result = runner.invoke(["--config-file", config_path, "connection", "list"])
 
-    assert len(caplog.messages) == 0
```

Comment:
Same

Reply:
So, we use weird text checking only because we don't trust snapshots?